src/nyx/scan_fastloader_way.cpp

```diff
@@ -121,6 +124,80 @@ namespace Nyxus
 		return true;
 	}
 
+#ifdef WITH_PYTHON_H
+	bool processIntSegImagePairInMemory (const py::array_t<unsigned int>& intens_fpath, const py::array_t<unsigned int>& label_fpath, int filepair_index, const std::string& intens_name, const std::string& seg_name)
+	{
+		std::vector<int> trivRoiLabels, nontrivRoiLabels;
+
+		// Timing block (image scanning)
+		{
+
+			{ STOPWATCH("Image scan2a/ImgScan2a/Scan2a/lightsteelblue", "\t=");
```

Comment:
STOPWATCH(.) relates to time measurement. We don't benchmark Python API functionality (too complex and secondary to CLI timing), we only do it for CLI